### ISSUE TITLE
Billing dataset or FOCUS dataset - terminology, definition and metadata requirements

### ISSUE THREAD

**Issue by ijurica** [FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec#472]
### Description
In the metadata-related chapters, we use the term "FOCUS dataset" 

>_... The FOCUS metadata schema column definition provides a list of the columns present in the FOCUS dataset along with metadata about the columns..._

In the rest of the specification, we usually refer to it as "billing data".
> _... column MUST be present in the billing data when ..._

I suggest we choose a consistent term and clearly specify it in the glossary. Additionally, we expect both FOCUS and custom/native (prefixed with x_) columns to be listed and documented in the schema, but I'm not sure if that's clear from the specification.

We often talk about the FOCUS (or billing) dataset as the cost and usage dataset, but the term "billing dataset" indicates that it's focused on billing and not usage, even though we have at least two usage-related columns and more are expected to come (this is based on the SKU details conversation where we mentioned that we are not only interested in attributes that affect the price/cost but also in details relevant for usage)

### Proposed approach
Choose a term (for instance, "FOCUS dataset"), specify it in the glossary, and use that term consistently throughout the specification.

**Comment by shawnalpay** [comment]
@ijurica, can you confirm that this issue is addressed by #592, and we can close this issue once that PR is merged?

**Comment by ijurica** [comment]
We've introduced the FOCUS dataset terminology, glossary term, and relevant links, but there's **one more point mentioned in this issue** that I'd like to discuss and hopefully address - **whether the inclusion of custom/native columns in the FOCUS dataset is recommended**.

@shawnalpay I've created and additional issue (Issue #602) for this purpose and suggest we close this one.

**Comment by shawnalpay** [comment]
Closing in favor of #602.

**Comment by jpradocueva** [comment]
### Action Items assigned during TF-2 call on Oct 15:
- [ ] [TF-1-[#472](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/issues/472)] Irena, @ijurica: Close Issue #472.
- [ ] [TF-1-#602] Irena, @ijurica: Create PR for updating the glossary to include provider-native columns

**Comment by jpradocueva** [comment]
### TF-2 call on Oct 15:
Topic: [#472](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/issues/472) Billing Data Set Terminology and Metadata Definition
Discussion Summary: This issue focused on defining terminology and metadata for the focus billing data set. The conversation highlighted that one specific point was left unresolved from the initial discussion, prompting a follow-up issue to address it.
Resolution: The team agreed to close issue #472, as the remaining point had been addressed in a new issue (#602), which will handle the outstanding discussion. The closure of #472 in favor of #602 was confirmed.
